odl/solvers/functional/default_functionals.py

```diff
@@ -118,6 +121,114 @@ def convex_conj(self):
         return IndicatorLpUnitBall(self.domain, exponent=np.inf)
 
 
+class GroupL1Norm(Functional):
+
+    """The functional corresponding to group L1-norm on `ProductSpace`s.
+
+    The L1-norm, ``|| ||x||_p ||_1``,  is defined as the integral/sum of
+    ``||x||_p``, where  ``||x||_p`` is the pointwise 2-norm.
+
+    This is also known as the cross norm.
+
+    Notes
+    -----
+    If the functional is defined on an :math:`\mathbb{R}^{n \\times m}`-like
+    space, the group :math:`L_1`-norm, denoted :math:`\| . \|_{\\times, p}` is
+    defined as
+
+    .. math::
+
+        \| x \|_{\\times, p} =
+        \\sum_{i = 1}^n \\left( \\sum_{j = 1}^m x_{i,j}^p \\right)^{1/p}.
+
+    If the functional is defined on an :math:`L_p^m`-like space, the group
+    :math:`L_1`-norm is defined as
+
+    .. math::
+
+        \| x \|_{\\times, p} =
+        \\int_{\Omega} \\left(\\sum_{j = 1}^m x_{j}^p\\right)^{1/p} dx.
```

Comment:
And by the way, why do you use `\times`? Why not `\|x\|_{1,2}`, indicating which norm in which "axis"?